Classes/genre.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'item'
+
+class Genre < Item
+  attr_accessor :name
+  attr_reader :items, :id
+
+  def initialize(name)
+    super()
```

Comment:
Hey Tracy, I have noticed that you have forgotten to initialize the `super() `as an `Item` constructor take on the parameter `publish_date` i think. Please consider it